perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -1200,13 +1237,49 @@ def ListConsistencyBenchmark(storage_schema, host_to_connect=None):
                final_result)
 
   # Final clean up: delete the objects still remaining.
-  DeleteObjectsConcurrently(storage_schema,
+  DeleteObjectsConcurrently(backend,
                             per_thread_objects_written,
-                            host_to_connect,
                             per_thread_objects_deleted)
   return final_result
 
 
+def TestBackend(backend):
```

Comment:
This is actually a poorly-named benchmark, not a backend implementation. I moved it to the script integration tests directory, and hopefully also improved the name.